source/Octopus.Server.Client/Model/TenantVariableResource.cs

```diff
@@ -14,6 +15,78 @@ public class TenantVariableResource : Resource, IHaveSpaceResource
         
         // Token to validate that no variables for the given tenant have changed since this TenantVariableResource was last requested.
         public string ConcurrencyToken { get; set; }
+
```

Comment:
Prior to this PR, the Resource types were simple bags of data that would be easy to pass around and perhaps mock. Adding the methods increases the API surface and seeing as though they all operate over the existing data structures and don't introduce any new data or encapsulation, I feel like they'd be better as Extension Methods on `TenantVariableResource`, etc.

Tests and percieved API surface would be unchanged, but it draws a bit of a line between the core thing and the convenience helpers.

Reply:
I was originally going to put these on the "editor" type for tenant variables. Looking at `TenantEditor` as an example, it just delegates to method calls on the resource itself. I was content with conforming to this existing pattern rather than trying to establish a new one.

Put simply, no new ground is being broken here.